editor.planx.uk/src/pages/FlowEditor/components/Flow/FeedbackPage.tsx

```diff
@@ -46,20 +90,216 @@ const getDetailedFeedback = async (feedbackId: number) => {
+  const displayFeedbackItems = [
+    "userComment",
+    "address",
+    "projectType",
+    "where",
+    "browserPlatform",
+  ];
+
+  return (
+    <Container maxWidth="contentWrap">
+      <Box py={4}>
+        <EditorRow>
+          <Typography variant="h2" component="h3" gutterBottom>
+            Feedback log
+          </Typography>
+          <Typography variant="body1">
+            Feedback from users about this service.
+          </Typography>
+        </EditorRow>
+        <EditorRow>
+          <Feed>
+            <Table stickyHeader sx={{ tableLayout: "fixed" }}>
+              <TableHead>
+                <TableRow
+                  sx={{ "& > *": { borderBottomColor: "black !important" } }}
+                >
+                  <TableCell sx={{ width: 160 }}>
+                    <strong>Type</strong>
+                  </TableCell>
+                  <TableCell sx={{ width: 100 }}>
+                    <strong>Date</strong>
+                  </TableCell>
+                  <TableCell sx={{ width: 340 }}>
+                    <strong>Comment</strong>
+                  </TableCell>
+                  <TableCell sx={{ width: 60 }}></TableCell>
+                </TableRow>
+              </TableHead>
+              <TableBody>
+                {feedback.map((item) => (
+                  <CollapsibleRow
+                    key={item.id}
+                    {...item}
+                    displayFeedbackItems={displayFeedbackItems}
+                  />
+                ))}
+              </TableBody>
+            </Table>
+          </Feed>
+        </EditorRow>
+      </Box>
+    </Container>
+  );
+};
+
+interface CollapsibleRowProps extends Feedback {
+  displayFeedbackItems: string[];
+}
+
+const CollapsibleRow: React.FC<CollapsibleRowProps> = (item) => {
+  const [open, setOpen] = useState<boolean>(false);
+  const [detailedFeedback, setDetailedFeedback] = useState<
+    Record<string, any> | undefined
+  >(undefined);
+  const { icon, title } = feedbackTypeIcon(item.type);
+
+  const toggleDetailedFeedback = async () => {
+    setOpen(!open);
+    if (!open && !detailedFeedback) {
+      const fetchedData = await getDetailedFeedback(item.id);
+      setDetailedFeedback(fetchedData);
+    }
+  };
+
+  const commentSummary = item.userComment
+    ? item.userComment.length > 50
+      ? `${item.userComment.slice(0, 50)}...`
+      : item.userComment
+    : "No comment";
+
+  const filteredFeedbackItems = (() => {
+    switch (item.type) {
+      case "issue":
+        return ["userContext", ...item.displayFeedbackItems];
+      case "helpful":
+      case "unhelpful":
+        return ["combinedHelp", ...item.displayFeedbackItems];
+      default:
+        return item.displayFeedbackItems;
+    }
+  })();
+
+  const labelMap: Record<string, string> = {
+    userComment: item.type === "issue" ? "What went wrong?" : "User comment",
+    address: "Project address",
```

Comment:
nit: I'm not sure we use "Project address" anywhere else - is "Property address" or "Site address" perhaps more consistent with service content ?

Reply:
I'll change to "Property address" as this matches the question "Enter the postcode of the property" where the data is gathered